cvmfs/cvmfs.cc

```diff
@@ -222,18 +222,20 @@ uint64_t next_directory_handle_ = 0;
 // contains inode to chunklist and handle to fd maps
 ChunkTables *chunk_tables_;
 
-perf::Statistics *statistics_;
-atomic_int64 num_fs_open_;
-atomic_int64 num_fs_dir_open_;
-atomic_int64 num_fs_lookup_;
-atomic_int64 num_fs_lookup_negative_;
-atomic_int64 num_fs_stat_;
-atomic_int64 num_fs_read_;
-atomic_int64 num_fs_readlink_;
-atomic_int64 num_fs_forget_;
-atomic_int32 num_io_error_;
-atomic_int32 open_files_; /**< number of currently open files by Fuse calls */
-atomic_int32 open_dirs_; /**< number of currently open directories */
+perf::Statistics *statistics_ = NULL;
+perf::Counter *n_fs_open_ = NULL;
+perf::Counter *n_fs_dir_open_ = NULL;
+perf::Counter *n_fs_lookup_ = NULL;
+perf::Counter *n_fs_lookup_negative_ = NULL;
+perf::Counter *n_fs_stat_ = NULL;
+perf::Counter *n_fs_read_ = NULL;
+perf::Counter *n_fs_readlink_ = NULL;
+perf::Counter *n_fs_forget_ = NULL;
+perf::Counter *n_io_error_ = NULL;
+/**< number of currently open files by Fuse calls */
```

Comment:
I think the Doxygen `/**<` syntax only works for comments that are in the same line.  For comments in front of a code line, the

    /**
     * comment
    */

syntax should be used.